---
summary: |
  This tutorial will introduce you to FlyBase ([flybase.org](http://flybase.org)).  FlyBase (short for fly database) is a website for the *Drosophila* scientific community that organizes research results about *Drosophila*.
author:
  - "[Katherine Cox](https://c-moor.github.io/portfolio/coxkatherine/)"
time: "20 min"
prerequisites: "None"
---

## Use FlyBase to investigate *labial*

The *labial* gene (gene symbol: *lab*) gene is a gene involved in the copper region of the *Drosophila* midgut, as well as controling the identity of segments that contribute to the head and the anterior thorax.  We can use FlyBase to learn more about this gene.

### Find the FlyBase ID
In most datasets, genes will be listed by an ID number rather than their name, such as the FlyBase ID number (FBgn number).  We can use Flybase to find these IDs and match them up to data in the dataset we're investigating.  Go to FlyBase and search for "lab" in the "Jump to Gene" searchbar.

```{r, flybase-lab, echo=FALSE}
quiz(caption = "Look up the *lab* gene on FlyBase",
  question("What is the FlyBase ID number for *lab*?",
    answer("FBgn0002522", correct=TRUE),
    answer("FBgn0000024"),  
    answer("FBgn0000003"),
    answer("FBgn0001264"),
    allow_retry = TRUE,
    random_answer_order = TRUE
  ),
  question("What biological processes is the *lab* gene involved in? HINT: Choose more than one answer.",
    answer("brain development", correct=TRUE),
    answer("midgut development", correct=TRUE),
    answer("immune system"),
    answer("cell cycle"),
    allow_retry = TRUE,
    random_answer_order = TRUE
  )
)
```

## Finding Orthologs

### Introduction

FlyBase also contains information about **orthologs** for each of the FlyBase genes.  Orthologs are genes that have similar sequences and functions across species.

Finding orthologs can be challenging - the sequences are similar, but not identical, and it can be hard to predict whether the gene is likely to be doing the same thing in other species.  There are several different computer programs that will analyze genes (using different features, including their nucleotide and amino acid sequences, predicted structures, and genome locations) to try to identify orthologs.

### DIOPT to the rescue!

With so many options for finding orthologs, how do you know which one to use?  Fortunately, we don’t have to choose.

DIOPT (DRSC Integrative Ortholog Prediction Tool) is a convenient tool that combines and summarizes information from several different ortholog prediction programs.  Rather than having to pick a single program, you can look at the DIOPT score and see how many of the programs think that a particular gene is an ortholog.

The DIOPT score is very straightforward: out of the total number of ortholog prediction programs, how many of them predicted this gene was an ortholog?  As of writing this tutorial, there are 15 programs for predicting fly-human orthologs, so a score of 7 means 7 out of those 15 programs think that the gene is an ortholog.

FlyBase uses DIOPT to report orthologs.  If you scroll down and expand the Orthologs section for a gene, you can see the list of potential orthologs and their DIOPT scores for several different species, including humans.  It also tells you which programs did or did not predict that particular ortholog.

Here are the top few predicted human orthologs for the *Ace* gene:
![FlyBase orthologs for the *Ace* gene](images/FlyBase_Ace_orthologs.png){width=100%}

In the results, "Hsap" stands for *Homo sapiens*, and the letters and numbers after the slash are the gene symbol.  You can click on these gene symbols to find out more about the human genes.

As you can see, the first two results (BCHE and ACHE) score quite well (10 out of 15 and 9 out of 15), so there’s a pretty good chance that they are orthologs.  It’s not uncommon for there to be multiple human genes that are orthologs of a single fly gene - humans have larger and more complex genomes, so they may have multiple specialized versions of a gene where flies only need a single version.  The rest of the results do not score very high and are much less likely to be true orthologs.

Once you know the likely human orthologs, you can then research those genes in other databases and scientific literature.  We can go back and forth between flies, humans, and other species and compare what we know across all species to try to build up an understanding of what the gene does and what happens when it stops working correctly.

```{r huntingtons-lab-orthologs}
quiz(caption = "Investigate orthologs for the *lab* gene",
  question("What is the most likely human ortholog for the *lab* gene?",
    answer("HTT"),
    answer("HOXA", correct = TRUE),
    answer("HSAP"),
    answer("LAT"),
    allow_retry = TRUE,
    random_answer_order = TRUE
  ),
  question("How many programs think this gene is an ortholog of *lab*?",
    answer("1"),
    answer("4"),
    answer("7", correct = TRUE),
    answer("11"),
    answer("15"),
    allow_retry = TRUE
  )
)
```

## Disease models

Because disease models are of particular interest, there is a special section in the FlyBase gene report about relationships between that FlyBase gene and any human diseases.  Note that there are 3 types of relationships that are reported:

- **Models Based on Experimental Evidence**: someone has studied flies with disruptions to this gene and has shown that the flies have similar symptoms to humans
- **Potential Models Based on Orthology**: we think this gene could be used to study a particular disease because we think it’s an ortholog, but we don’t have proof.
- **Modifiers Based on Experimental Evidence**: this gene doesn’t directly cause a disease, but makes a disease caused by a different gene better or worse

### Disease report for the *Ace* gene

The *Ace* gene has no known or predicted relationship to any human diseases, so its report is pretty boring in this section:

![FlyBase disease information for the *Ace* gene.  There are zero disease associations.](images/FlyBase_Ace_disease.png){width=100%}

### Huntington’s Disease

Let’s take a look at something more interesting.  Say we've heard that scientists are using fruit flies to study Huntington’s disease.  Can we use FlyBase to find out more?

FlyBase has lots of information besides just reports about specific genes.  If you go to the FlyBase homepage and scroll down to the Quick Search box, you can see some of the categories of information available.  Let’s search for the Huntington’s disease model:

1. Go to the FlyBase homepage
1. In the Quick Search box, select "Human Disease"
1. Enter "Huntington’s disease"

![Screenshot of FlyBase Quick Search](images/FlyBase_QuickSearch.png){width=100%}

This will give you a list of everything related to Huntington’s disease on FlyBase:

![FlyBase search results for Huntington's disease](images/FlyBase_search_Huntingtons.png){width=100%}


At the top we see:
The **Human Disease Model** report - information about models for this disease
The **Disease Ontology** report - information from a database about human diseases
The **Gene Report** for the human (*Homo sapiens*) gene: Hsap/HTT
The **Gene Report** for the *Drosophila* homolog: Dmel/htt

The Human Disease Model report is a good place to start.  Clicking on this result takes us to the [full report](https://flybase.org/reports/FBhh0000003).

At the top we can see some information about the disease, including a brief overview of the disease and a link to the OMIM (Online Mendelian Inheritance in Man) database entry for the disease:

![FlyBase Human Disease Model report for Huntington's disease](images/FlyBase_Huntingtons.png){width=100%}


Further down the page, there is a section describing the relevant *Drosophila* gene(s):

![Drosophila gene information from the Human Disease Model report for Huntington's disease](images/FlyBase_Huntingtons_Drosophila_genes.png){width=100%}

```{r huntingtons-quiz-1}
quiz(caption = "Explore the Huntington's disease report",
  question("What is the gene symbol for the Drosophila ortholog of the human Huntington's disease gene?",
    answer("hunt"),
    answer("HD"),
    answer("htt", correct = TRUE),
    answer("Dmel"),
    allow_retry = TRUE,
    random_answer_order = TRUE
  )
)
```

### Investigate the *htt* gene

Now that we know the *htt* gene is used to model Huntington’s disease, let’s find out more about this gene.  Look up the Gene report for this gene.  The easiest way to do this is to type "htt" in the Jump to Gene box.

```{r huntingtons-quiz-2}
quiz(caption = "Explore the *htt* gene report",
  question("What biological processes is the *htt* gene involved in? HINT: Choose more than one answer.",
    answer("brain development"),
    answer("choline catabolic process"),
    answer("establishment of mitotic spindle orientation", correct = TRUE),
    answer("synaptic vesicle transport", correct = TRUE),
    answer("cell cycle"),
    answer("axo-dendritic transport", correct = TRUE),
    allow_retry = TRUE,
    random_answer_order = TRUE
  ),
  question("How many different *alleles* (versions of the gene) can be used as models of Huntington's disease, **based on experimental evidence**?",
    answer("0"),
    answer("1"),
    answer("2"),
    answer("3", correct = TRUE),
    answer("4"),
    allow_retry = TRUE
  )
)
```

### *htt* alleles (variant forms of the gene)

Yes!  Here we can see that there are actually three different variants of the *htt* gene that have been experimentally shown to cause symptoms similar to Huntington’s disease in flies.  These variants all have different types of mutations affecting the *htt* gene.

It can be useful to have different versions of the gene that all mimic Huntington's disease.  Depending how these mutations were made or discovered, they can be useful for different types of experiments.  Also, if experiments turn out the same for several different disease-causing mutations, it's more likely that the result will transfer to other organisms, and isn't just a quirk of a particular mutation.

## Summary

FlyBase is a fantastic source of information.  You can use FlyBase to find:

- Gene functions and interactions
- Orthologs (*Drosophila* genes that are related to genes in other species, including humans)
- *Drosophila* genes that are relevant to human diseases
- The name, symbol or ID number for a gene, so you can research it on other databases.

#### Learn More

FlyBase is designed to help scientists do research, so it has *a lot* of information, and we've just looked at a few of the basic features.  Here are some resources if you'd like to learn more:

+ Check out the "Help" tab on FlyBase
+ Visit [FlyBase TV](https://www.youtube.com/c/FlyBaseTV), the FlyBase YouTube channel
+ Look at the FlyBase "[New to Flies](https://wiki.flybase.org/wiki/FlyBase:New_to_Flies?)" wiki page
